src/pkgcli.h

```diff
@@ -30,6 +30,7 @@
 
 #include <stdint.h>
 #include <bsd_compat.h>
+#include "external/libelf/_elftc.h"
```

Comment:
STAILQ doesn't seem to exist in `sys/queue.h` and isn't mentioned in the `QUEUE(3)` man page. OpenBSD has TAILQ though.